samples/net/updatehub/src/c_certificates.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019 O.S.Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __CERTIFICATE_H__
+#define __CERTIFICATE_H__
+
+#define CA_CERTIFICATE_TAG 1
+
+static const unsigned char server_certificate[] = {
```

Comment:
Again, why is this in a header file? The code and data should be in C files.

Reply:
I followed an example provided here for testing with dtls, in this example (https://github.com/zephyrproject-rtos/zephyr/tree/master/samples/net/sockets/echo_client/src) the certificates were used in .h file, so I thought this was the default template for setting certificates.

Reply:
I'm not sure why that sample is in a header file, either. I'm just thinking from a code point of view. You should assume a header file will be included in multiple places, otherwise there really isn't much point in putting it in a header file.

Reply:
Right but this needs to be consistent on the whole project. I am keen to keep this as is for now and we can fix it globally next. What do you think?